packs/aws/config.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 setup:
-  region: ""
-  aws_access_key_id: ""
-  aws_secret_access_key: ""
+  region: "us-east-1"
```

Comment:
No sure if you intended to check-in `us-east-1`. Typically the config should be empty unless the default value is unambiguous :)